app/src/common/shared/org/mozilla/vrbrowser/VRBrowserActivity.java

```diff
@@ -969,8 +969,11 @@ void resumeGeckoViewCompositor() {
         GleanMetricsService.stopImmersive();
         Handler handler = new Handler(Looper.getMainLooper());
         handler.postDelayed(() -> {
-            mWindows.resumeCompositor();
-            Log.d(LOGTAG, "Compositor Resumed");
+            if (!mWindows.isPaused()) {
+                Log.d(LOGTAG, "Compositor resume begin");
+                mWindows.resumeCompositor();
+                Log.d(LOGTAG, "Compositor resume end");
+            }
```

Comment:
These logs should probably use the `LOGTAG`

Reply:
done